web/source/osk/oskManager.ts

```diff
@@ -196,10 +196,7 @@ namespace com.keyman.osk {
         }
         // For other devices, adjust the object heights, allowing for viewport scaling
       } else {
-        this.vkbd.refreshLayout(this.getKeyboardHeight());
-
-        var b: HTMLElement = this._Box, bs=b.style;
-        bs.height=bs.maxHeight=this.vkbd.computedAdjustedOskHeight(this.getHeight())+'px';
+        this.refreshLayout();
       }
```

Comment:
Turns out this pattern existed in a few places in a consistent manner - this is how the `OSKManager` has been setting the size of the _touch_ OSK and adjusting its layout.  That operation is part of how the `OSKManager`'s layout itself is managed... thus makes a good starting point for a `OSKManager`-level `refreshLayout`.